src/main/java/luke/tasks/Deadline.java

```diff
@@ -0,0 +1,30 @@
+package luke.tasks;
+
+public class Deadline extends Task{
+    protected String by;
+
```

Comment:
There is an excessive use of whitespace here.
(This is part of the intermediate portion of coding standard.)